modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectPatternConfigurationDisplayContext.java

```diff
@@ -58,6 +58,8 @@ public String getRedirectPatternConfigurationURL() {
 
 	public Map<String, Object> getRedirectPatterns() {
 		return HashMapBuilder.<String, Object>put(
+			"actionUrl", getRedirectPatternConfigurationURL()
```

Comment:
I could not found another place than here that use : `getRedirectPatternConfigurationURL`
Suggestion: 
Maybe change it to be private is better?